src/components/SingleSelect/SingleSelect.jsx

```diff
@@ -174,7 +180,8 @@ const SingleSelect = createClass({
 			>
 				<DropMenu.Control>
 					<div className={cx('&-Control', {
-						'&-Control-is-selected': !isDisabled && isItemSelected,
+						'&-Control-is-highlighted': (!isDisabled && isItemSelected && isSelectionHighlighted) || (isExpanded && isSelectionHighlighted),
```

Comment:
renamed it from `-is-selected` to `-is-highlighted` but kept the former for backward compatibility. Have an issue with 2.0.0 milestone to deprecate it: https://github.com/appnexus/lucid/issues/289